datafusion/physical-plan/src/sorts/sort.rs

```diff
@@ -1064,6 +1076,7 @@ mod tests {
         assert!(metrics.elapsed_compute().unwrap() > 0);
         assert!(metrics.spill_count().unwrap() > 0);
         assert!(metrics.spilled_bytes().unwrap() > 0);
+        assert!(metrics.spilled_rows().unwrap() > 0);
```

Comment:
I wonder if we can actually check the spilled row count here 🤔 It should be deterministic right?

Reply:
Yes, these metrics results are deterministic so added for `spill_count` and `spilled_bytes` in addition to `spilled_rows`.